src/app/static/src/app/components/outputTable/TableReshapeData.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+    <table-display :data="reshapedData" :header-name="tableLabelHeader"/>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent } from 'vue';
+import TableDisplay from './TableDisplay.vue';
+import { useStore } from 'vuex';
+import { RootState } from '../../root';
+import { Filter, FilterOption } from '../../generated';
+
+export default defineComponent({
+    components: {
+        TableDisplay
+    },
+    props: {
+        data: Object
+    },
+    setup(props) {
+        const store = useStore<RootState>();
+
+        const features = computed(() => store.state.baseline.shape?.data.features);
+        const selections = computed(() => store.state.plottingSelections.table);
+        const filterDataIdtoId = computed(() => {
+            const filters: Filter[] = store.getters["modelOutput/tableFilters"] || [];
+            const dataIdToId: Record<string, string> = {};
+            filters.forEach(f => {
+                dataIdToId[f.column_id] = f.id;
+            });
+            return dataIdToId;
+        });
+        
+        const presetMetadata = computed(() => {
+            const currentPreset = store.state.plottingSelections.table.preset;
+            const presetMetadata = store.state.modelCalibrate.metadata?.tableMetadata.presets;
+            const metadata = presetMetadata?.find(m => m.defaults.id === currentPreset);
+            if (currentPreset && presetMetadata && metadata) {
+                return { row: metadata.defaults.row, column: metadata.defaults.column };
+            }
+            return null;
+        });
+
+        const reshapedData = computed(() => {
+            const data = props.data;
+            if (presetMetadata.value && data) {
+                const reshapedData: any[] = [];
```

Comment:
It's a bit confusing having the local and computed name the same here.

Reply:
yep changed it to `tableData` for now? not sure I am not very good at naming these things haha